pages/12.Plugins/14.Salesforce/docs.en.md

```diff
@@ -262,16 +261,14 @@ Follow [these steps][testing] to test the integration.
 This means the API is not turned on in your Salesforce account. [Read more][salesforce-admin]
 
 
-[Salesforce CRM]: <https://www.salesforce.com/>
+[salesforce-crm]: <https://www.salesforce.com/>
 [salesforce-admin]: <https://help.salesforce.com/apex/HTViewHelpDoc?id=admin_userperms.htm&language=en>
-
+[official-documentation]:<http://feedback.uservoice.com/knowledgebase/articles/235661-get-your-key-and-secret-from-salesforce>
```

Comment:
Needs a space after the : 

```suggestion
[official-documentation]: <http://feedback.uservoice.com/knowledgebase/articles/235661-get-your-key-and-secret-from-salesforce>
```

Reply:
I didn't see this, thanks for picking this up